### ISSUE TITLE
`import/no-unresolved` should find `json` files

### ISSUE THREAD

**Issue by jfmengels** [benmosher/eslint-plugin-import#333]
I'm trying to add the plugin to a repo I'm working on, and I'm getting false positives with `no-unresolved` when importing JSON files.

```js
import setEvent from './fixtures/foo'; // <-- `./fixtures/foo.json` is an existing and valid JSON file
```

I'm not aware of it being incorrect to `import` a JSON file, but please let me know if that's the case.

**Comment by benmosher** [comment]
Weird. Assuming resolve supports it, I think it should. Sounds like a bug.

**Comment by shark0der** [comment]
I'm having a similar issue but with jsx files. Webpack doesn't complain at all.

**Comment by benmosher** [comment]
@jfmengels I just added tests but they are passing. Can you take a look? If you can construct a failing test I can patch it, but I'm not sure what the issue is. Maybe something with your config?

**Comment by jfmengels** [comment]
Hmm... Yeah, I'll try to take a look at it tonight.

**Comment by benmosher** [comment]
No rush, speed is up to you. AFAICT it's working. (also, sorry it took so long for me to look at this!)

**Comment by benmosher** [comment]
ah, yeah, so for this, `import/extensions` is the correct solution ATM: https://github.com/benmosher/eslint-plugin-import#importextensions

I think that should resolve it. See note in README about making `['.js']` the default in v2.

**Comment by benmosher** [comment]
Also, I'm confused why you don't have push rights to a branch, I thought you could push anywhere in the repo?

**Comment by jfmengels** [comment]
> Also, I'm confused why you don't have push rights to a branch, I thought you could push anywhere in the repo?

Fixed it, my bad. Didn't configure my git remote stuff well. Just pushed my `issue-333` branch over yours now.

And the `import/extensions` tip works :+1:

**Comment by benmosher** [comment]
Ah, wild. This is actually an "issue" with `node-resolve`. By default it only finds `.js` files.

I just need to add `.json` to the Node resolver and this should work.

**Comment by jfmengels** [comment]
:smile:

**Comment by benmosher** [comment]
Just published with node resolver v0.2.1
